internal/crypto/null_aead_fnv128a.go

```diff
@@ -21,21 +22,21 @@ func (n *nullAEADFNV128a) Open(dst, src []byte, packetNumber protocol.PacketNumb
 		return nil, errors.New("NullAEAD: ciphertext cannot be less than 12 bytes long")
 	}
 
-	hash := fnv128a.New()
+	hash := fnv.New128a()
 	hash.Write(associatedData)
 	hash.Write(src[12:])
 	if n.perspective == protocol.PerspectiveServer {
 		hash.Write([]byte("Client"))
 	} else {
 		hash.Write([]byte("Server"))
 	}
-	testHigh, testLow := hash.Sum128()
+	sum := make([]byte, 0, 16)
+	hash.Sum(sum)
+	sum = sum[:16]
```

Comment:
This should be unnecessary.

Reply:
It's not if we use the return value. We should do that.